lib/ash/type/type.ex

```diff
@@ -1638,6 +1638,11 @@ defmodule Ash.Type do
 
         @impl true
         def embed_as(_, _), do: :self
+
+        @impl true
+        def autogenerate(_) do
+          Enum.at(@parent.generator([]), 0)
```

Comment:
🤔 this is an interesting implementation. For many types this wouldn't really work, like for strings it would return empty strings etc.

Is this *only* used in ecto when generating a type when explicitly configured with that module attribute?

If so, then I'm okay with this implementation.

However, the first argument is likely the constraints which need to be passed through to `generator/1`

Reply:
`Enum.at` is the right way, the main thing I'm wondering is if we want to make this an option in the type whether or not to define this callback. `use Ash.Type, autogeneratable?: true`

Reply:
LGTM!